src/app/data-planes/components/DataPlaneList.vue

```diff
@@ -205,15 +198,18 @@ type ChangeValue = {
 }
 
 const props = withDefaults(defineProps<{
-  total?: number,
-  pageNumber: number,
-  pageSize: number,
-  items: DataPlaneOverview[] | undefined,
-  error: Error | undefined,
+  total?: number
+  pageNumber: number
+  pageSize: number
+  items: DataPlaneOverview[] | undefined
+  error: Error | undefined
```

Comment:
I very often see these `,` changes between PRs which makes me think that @johncowen and you don't have the same editor config. Can we add a linter so we stop changing these back and forth (linter has the advantage that it will work for future folks too)? This would preserve a clean git history and make PR reviews simpler

Reply:
Ah yeah this is my bad, would definitely be good to get a lint rule in there. I'd prefer type definitions without commas, but habit/muscle memory got the better of me here it seems.

Reply:
We have linting setup for this, but it doesn’t seem to work for JavaScript expressions that are inside Vue templates. Perhaps that’s an easy fix.

Reply:
(And this change is in line with our linter configuration)

Reply:
Oh, I realized that I’ve commented on the wrong thread. My two replies here should’ve been made here https://github.com/kumahq/kuma-gui/pull/1646#discussion_r1371682731:

As for this, this specifically happens inside TypeScript types where a delimiter (both comma and semi-colon are allowed) is always optional. We don’t have a linter configuration for this. Not sure if there is a rule. If there is, we can standardize this .

Reply:
This issue will be fixed/standardized here: https://github.com/kumahq/kuma-gui/pull/1673